package.json

```
@@ -61,14 +61,14 @@
     "trailingComma": "all"
   },
   "dependencies": {
-    "@babel/runtime": "^7.4.2",
+    "@babel/runtime": "^7.13.8",
     "@restart/context": "^2.1.4",
-    "@restart/hooks": "^0.3.21",
+    "@restart/hooks": "^0.3.26",
     "@types/classnames": "^2.2.10",
     "@types/invariant": "^2.2.33",
     "@types/prop-types": "^15.7.3",
-    "@types/react": ">=16.9.35",
-    "@types/react-transition-group": "^4.4.0",
+    "@types/react": ">=17.0.2",
```

```suggestion
    "@types/react": ">=16.9.35",
```

We should change this back to allow 16.x types?

---

oops